modules/standard/CommDiagnostics.chpl

```diff
@@ -254,9 +254,33 @@ module CommDiagnostics
      */
     var execute_on_nb: uint(64);
 
+    /*
+
+      GETs that were handled by the cache. GETs counted here did not require
+      the cache to communicate in order to return the result.
+
+      .. note:
+
+        A GET that requires the cache to wait (but not create any other
+        communication) is counted here. As a result, a GET that needed to wait
+        for a prefetch to complete is considered a hit.
+
+     */
     var cache_get_hits: uint(64);
+
+    /*
+      GETs that were not handled by the cache - that is, GETs where the
+      cache needed to communicate in order to handle the GET.
```

Comment:
This one has too many "GET"s. maybe end with "... cache needed to communicate with another locale"?